08-projects.Rmd

```diff
@@ -6566,18 +6566,18 @@ The following questions will use the dataset found in Scholar:
 
 `/class/datamine/data/movies_and_tv/imdb.db`
 
-In this project we want to offer the flexibility of using your choice of R and/or Python. To keep things as consistent as possible, please use Rmarkdown on https://rstudio.scholar.rcac.purdue.edu/. See [here](#python-on-scholar) to learn how to run Python in this environment.
-
 #### Questions 
 
 ##### 1. What is the first year where our database has > 1000 titles? Use the `premiered` column in the `titles` table as our year. What year has the most titles?
 
-**Relevant topics:** [COUNT](#sql-examples), [GROUP BY](#sql-groupby), [ORDER BY](#sql-order-by), [DESC](#sql-examples)
+Hint: There could be missing values in `premiered`. We want to avoid them in SQL queries, for now. You can learn more about the missing values (or NULL) in SQL [here](https://www.w3schools.com/sql/sql_null_values.asp).
```

Comment:
I added this hint following just like Question 3 in Project 12 to avoid missing values.

Reply:
Good idea!